### ISSUE TITLE
AUDIT - Context Menu Zoom

### ISSUE THREAD

**Issue by trallard** [Quansight-Labs/jupyterlab-accessible-themes#14]
- [ ] 200%
- [ ] 400%
- [ ] Change the default value of text size from 16 to 30 in the web browser
- [ ] Test with different screen sizes

## References

- [Theme scope checklist](https://github.com/isabela-pf/accessibility/pull/1/files#diff-084cbb6818caeb52700399405bb8c8e8ed366445ad27c7ac83c70aa10f519382)
- [Audit spreadsheet](https://docs.google.com/spreadsheets/d/1yaWY--2zBi7KqPAnMMbJp_aVXFFT8sBDdci8UfpKjO4/edit#gid=0)

**Comment by steff456** [comment]
These are the results of the Zoom Audit,

## Context menu behavior

<img width="609" alt="image" src="https://user-images.githubusercontent.com/20992645/183738423-c4887119-cfd8-4153-a69c-d7b11337e1c1.png">

## Notes
- All context menus have the same class assigned to them in css `lm-Menu p-Menu`
- The options in the context menu vary depending on where in the interface it is clicked
- Automatically we have a scrollbar to navigate through the options if not visible at first
- The size of the context menu size is fixed in px
- Some guidelines state that there should be a visible button to open the context menu - in the case that we want to fully navigate with keyboard and to guarantee that screen readers can open it
- Changing the default text size doesn't do any effect in Jlab UI, but changing the minimum text size in the browser does create changes.

## Suggestions
- Check the UI/UX for context menus in small screens or high zoom
- Do we need to add a button so the context menus can open without having to right click them?
